packages/blade/src/components/TextArea/_decisions/decisions.md

```diff
@@ -0,0 +1,66 @@
+| placeholder          | `string`                         |         | Placeholder text to be displayed inside the text area                                                                                                                                                                                                        |          |
+| value                | `string`                         |         | The value of the TextArea, if present will act as [controlled component](https://reactjs.org/docs/forms)                                                                                                                                                     |          |
+| defaultValue         | `string`                         |         | The initial value of the TextArea                                                                                                                                                                                                                            |          |
+| name                 | `string`                         |         | The name of the input, useful in form submissions                                                                                                                                                                                                            |          |
+| onChange             | `({ event, name, value }) => {}` |         | Function called when the value of the input field changes                                                                                                                                                                                                    |          |
+| onBlur               | `({ event, name, value }) => {}` |         | Function called when the input loses focus                                                                                                                                                                                                                   |          |
+| isDisabled           | `boolean`                        | `false` | Marks the input field as disabled. It'll also set aria-disabled to true and hence the input field will loose keyboard focus                                                                                                                                  |          |
+| isRequired           | `boolean`                        | `false` | Marks the input field as required. if the errorText is provided then that will be shown if the field is required and not filled, else ${label} is required                                                                                                   |          |
```

Comment:
do we really need to show the `${label} is required`? also don't think this is supported by our `FormHint` component yet

Reply:
Sorry this line is a bit confusing I guess. What i meant was that the input will be set to `aria-required/required` prop. 
Visually we handle the required prop via `neccessityIndicator`